packages/monorobot/monorobot.0.1/opam

```diff
@@ -0,0 +1,37 @@
+opam-version: "2.0"
+synopsis: "Notification bot for monorepos"
+description: "Notification bot to handle webhook events and post notifications to Slack."
+maintainer: "Ahrefs <github@ahrefs.com>"
+authors: "Ahrefs <github@ahrefs.com>"
+homepage: "https://github.com/ahrefs/monorobot"
+bug-reports: "https://github.com/ahrefs/monorobot/issues"
+dev-repo: "git+https://github.com/ahrefs/monorobot.git"
+depends: [
+ "ocaml" {>= "4.03.0"}
+ "dune" {>= "2.5.0"}
+ "atdgen"
+ "base"
+ "base64"
+ "biniou"
+ "cmdliner"
+ "cstruct"
+ "devkit"
+ "hex"
+ "lwt"
+ "lwt_ppx"
+ "nocrypto"
+ "re2"
+ "stdio"
+ "uri"
+ "ocamlformat" {dev}
```

Comment:
```suggestion
```
I’m not seeing any use of ocamlformat during the compilation. If what you want is to automatically install what you use personally you can always have a custom makefile rule or similar other script to install everything you use.